Documentation/sources/enum-push-tracking-status.md

```diff
@@ -1,25 +0,0 @@
-# PushTrackingStatus
-
-Enum representing the status of push tracking.
-
-### Definition
-
-```java
-@objc(AEPPushTrackingStatus)
-public enum PushTrackingStatus: Int {
```

Comment:
This value is returned in the optional callback passed to public API `handleNotificationResponse`. Can we keep it?